src/nft/pages/collection/CollectionDesktop.tsx

```diff
@@ -0,0 +1,33 @@
+import { AnimatedBox, Box } from 'nft/components/Box'
+import { CollectionStats } from 'nft/components/collection/CollectionStats'
+import { Column, Row } from 'nft/components/Flex'
+import { CollectionProps } from 'nft/pages/collection/common'
+import * as styles from 'nft/pages/collection/common.css'
+import { ActivitySwitcher } from 'nft/pages/collection/Components'
+
+export const CollectionDesktop = ({ collectionStats, isActivityToggled }: CollectionProps) => {
+  return (
+    <Column width="full">
+      <Box width="full" height="160">
+        <Box
+          as="img"
+          maxHeight="full"
+          width="full"
+          src={collectionStats?.bannerImageUrl}
+          className={`${styles.bannerImage}`}
+        />
+      </Box>
+
+      <Row paddingLeft="32" paddingRight="32">
+        <CollectionStats stats={collectionStats} isMobile={false} />
+      </Row>
+      <Row paddingLeft="32" paddingRight="32">
+        <ActivitySwitcher showActivity={isActivityToggled} toggleActivity={() => undefined} />
+      </Row>
+      <Row alignItems="flex-start" position="relative" paddingLeft="32" paddingRight="32">
+        {/* // @ts-ignore */}
```

Comment:
What is being ignored here? Can this line be removed?

Reply:
see above

Reply:
I don't see that locally. Can you try removing it and see if the CI fails?